SD-K9/src/app/pages/bus/bus.page.spec.ts

```diff
@@ -16,9 +20,24 @@ describe('BusPage', () => {
     fixture = TestBed.createComponent(BusPage);
     component = fixture.componentInstance;
     fixture.detectChanges();
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000000;
```

Comment:
This line could block a test run for about 17 minutes should something block one of the tests. I'd suggesting setting the timeout to something along the lines of 30 seconds.

Reply:
thanks for pointing that out, i didn't notice it was that long